vic/drivers/image/src/vic_init_output.c

```
@@ -193,10 +208,212 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining time dimenension in %s", nc->fname);
     }
 
+    // define the netcdf variable time
+    status = nc_def_var(nc->nc_id, "time", NC_DOUBLE, 1,
+                        &(nc->time_dimid), &(time_var_id));
+    if (status != NC_NOERR) {
+        log_err("Error defining time variable in %s", nc->fname);
+    }
+    status = nc_put_att_text(nc->nc_id, time_var_id, "standard_name",
+                             strlen("time"), "time");
+    if (status != NC_NOERR) {
+        log_err("Error adding attribute in %s", nc->fname);
+    }
+
+    // adding units attribute to time variable
+    if (global_param.time_units == TIME_UNITS_SECONDS) {
+        sprintf(strUnit, "seconds");
+    }
+    else if (global_param.time_units == TIME_UNITS_MINUTES) {
+        sprintf(strUnit, "minutes");
+    }
+    else if (global_param.time_units == TIME_UNITS_HOURS) {
+        sprintf(strUnit, "hours");
+    }
+    else if (global_param.time_units == TIME_UNITS_DAYS) {
+        sprintf(strUnit, "days");
+    }
+    else {
+        sprintf(strUnit, "-");
```

better to raise an error here if we don't know what the time units are.